addons/account_payment_term/tests/test_payment_term.py

```diff
@@ -0,0 +1,53 @@
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.tests import tagged
+from odoo import Command, fields
+from odoo.tests.common import Form
+
+
+@tagged('post_install', '-at_install')
+class TestAccountPaymentTerms(AccountTestInvoicingCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.invoice = cls.init_invoice('out_refund', products=cls.product_a + cls.product_b)
+
+        cls.pay_term_a = cls.env['account.payment.term'].create({
+            'name': "basic case",
+            'line_ids': [
+                Command.create({
+                    'value': 'percent',
+                    'value_amount': 100,
+                    'nb_days': 30,
+                    'delay_type': 'days_end_of_month_on_the',
+                    'days_next_month': 10,
+                }),
+            ],
+        })
+        cls.pay_term_b = cls.env['account.payment.term'].create({
```

Comment:
What about using meaningful variable name instead of `pay_term_a` and `pay_term_b`? 
Like `pay_term_days_end_of_month_10` and `pay_term_days_end_of_month_31`?